velox/functions/prestosql/Arithmetic.h

```diff
@@ -495,5 +499,23 @@ struct TruncateFunction {
   }
 };
 
+template <typename T>
+struct BetaCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double a, double b, double value) {
+    VELOX_USER_CHECK((a > 0) && (b > 0), "alpha and beta must be > 0");
+    VELOX_USER_CHECK(
+        (value >= 0) && (value <= 1), "value must be in the interval [0, 1]");
+    VELOX_USER_CHECK(
+        (a != kInf) && (b != kInf),
+        "alpha, beta values can't accept infinity value");
```

Comment:
Presto seems to allow that:

```
presto:di> select beta_cdf(0.5, infinity(), 0);
 _col0
-------
   0.0
(1 row)
```

Reply:
Using boost library for this, which doesn't support infinity value for beta_cdf. I have added an if condition to add this.